rules_java_gapic/java_gapic.bzl

```diff
@@ -14,6 +14,7 @@
 
 load("@com_google_api_codegen//rules_gapic:gapic.bzl", "proto_custom_library", "unzipped_srcjar")
 
+SERVICE_YAML_ALLOWLIST = ["cloudkms"]
```

Comment:
Nit: WDYT about adding a TODO to remove this? Really not necessary, but might be nice documentation.

Reply:
Done at the usage site.